src/fetcher.ts

```diff
@@ -3,9 +3,9 @@ import { getNetwork } from '@ethersproject/networks'
 import { getDefaultProvider } from '@ethersproject/providers'
 import { TokenAmount } from './entities/fractions/tokenAmount'
 import { Pair } from './entities/pools'
-import IPangolinPair from '@pangolindex/exchange-contracts/artifacts/contracts/pangolin-core/PangolinPair.sol/PangolinPair.json'
 import invariant from 'tiny-invariant'
 import ERC20 from './abis/ERC20.json'
+import IPangolinPair from './abis/Pair.json'
```

Comment:
In this file there is issue at line 67 due to this change. Now we don't have `abi` field in `IPangolinPair` so its throwing error.